### DISCUSSION TITLE
ive been accepted for git hub student developer pack but cant receive my benefits

### DISCUSSION THREAD

**Discussion by quhtani-x** [community/community#76531] [GitHub Education]
### Hi everyone,

i got accepted for github student dev pack , when i try to receive my benefits i just keep being redirected in a loop ,

  
![image](https://github.com/community/community/assets/146584473/1a4b79fa-2110-4f73-b2a2-39d72a317395)

![image](https://github.com/community/community/assets/146584473/8016caf6-12bd-4eb2-b9a1-8ecfdb3226f5)
 
 when i click on the i'm a student button i get redirected again to the first page 
 
![image](https://github.com/community/community/assets/146584473/a7d40973-9328-4c03-8587-f14de0c90378)

**Comment by KeithPrinkey-ops** [comment]
I am having the same issue!

**Comment by ddddami** [comment]
Hii @quhtani-x @KeithPrinkey-ops login on github education dashboard, click on your profile icon, then 'your benefits'
![Screenshot (60)](https://github.com/community/community/assets/82976159/a42af5c5-ec14-4c87-b6b3-6322d424327e)
Click on the 'explore more offers' button
![Screenshot (61)](https://github.com/community/community/assets/82976159/d46b0b42-f536-4138-809f-bb7b865f43eb)
the first section you would see is about experiences. 
![Screenshot (62)](https://github.com/community/community/assets/82976159/5d348e9a-83c4-4ea6-aa65-b731947d928b)
scroll way past these.
until you get to the next section, 'all offers', here you can easily use your benefits
![Screenshot (63)](https://github.com/community/community/assets/82976159/af12a3f0-f00a-486b-a825-9223376d6cf9)

**Reply by KeithPrinkey-ops** [reply]
Thank you.

**Comment by samus-aran** [comment]
Hi @quhtani-x  👋

We're sorry you're experiencing this. Can we please ask you to try now and re-apply. We've raised this experience with our internal team but we hope this helps you proceed to re-verify your academic status. 👍

**Comment by KeithPrinkey-ops** [comment]
It worked just fine. I scrolled all the way down after clicking my benefits
and sure enough, there were links to the products. Have to merely sign in
with my your GitHub account @quhtani-x

**Comment by kanduru-abhiram** [comment]
Hey @quhtani-x, please check out below link for a detailed list of all the benefits covered under the Dev pack and do check the background color of Approved in the Application status. Check below image as reference.

![image](https://github.com/community/community/assets/85011280/4ce3be40-f28e-4d14-a74c-bcceef773024)

Benefits Link: https://education.github.com/pack/offers

Please accept this as a solution of helped, thanks
